From linux-usb-devel-admin@lists.sourceforge.net Wed Jan 18 13:16:04 2006
From: Alexey Dobriyan <adobriyan@gmail.com>
Message-ID: <20060118210607.GL12771@mipter.zuzino.mipt.ru>
Content-Disposition: inline
Subject: USB: arm26: fix compilation of drivers/usb/core/message.c
Date: Thu, 19 Jan 2006 00:06:07 +0300

drivers/usb/core/message.c:395: error: invalid use of undefined type `struct scatterlist'

Signed-off-by: Alexey Dobriyan <adobriyan@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/usb/core/message.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/core/message.c
+++ gregkh-2.6/drivers/usb/core/message.c
@@ -13,6 +13,7 @@
 #include <linux/ctype.h>
 #include <linux/device.h>
 #include <asm/byteorder.h>
+#include <asm/scatterlist.h>
 
 #include "hcd.h"	/* for usbcore internals */
 #include "usb.h"
